agent/api/task/task_linux.go

```diff
@@ -58,11 +56,11 @@ const (
 	ExecCommandAgentSessionWorkerBinName = "ssm-session-worker"
 
 	// TODO: [ecs-exec] decide if this needs to be configurable or put in a specific place in our optimized AMIs
```

Comment:
what's the status of this TODO?

Reply:
To be defined, the ssm agent is still working on this, so I want to make sure I don't forget by leaving this TODO here